lib/content-services/src/lib/document-list/components/file-auto-download/file-auto-download.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="adf-non-responsive-dialog-container">
```

Comment:
you need to use standard Mat Dialog elements instead of custom layout. Please check the docs https://material.angular.io/components/dialog/overview on "mat-dialog-title", "mat-dialog-body", etc.